server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/PreCheckRuntimeFlowableTask.java

```diff
@@ -236,26 +255,20 @@ private boolean isIntercepted(SqlCheckTaskResult sqlCheckResult,
         return false;
     }
 
-    private void doDatabasePermissionCheck() {
-        Set<String> unauthorizedDatabaseNames =
-                databaseService.filterUnAuthorizedDatabaseNames(
-                        SchemaExtractor.listSchemaNames(
-                                this.databaseChangeRelatedSqls.stream().map(OffsetString::getStr).collect(
-                                        Collectors.toList()),
-                                this.connectionConfig.getDialectType()),
-                        connectionConfig.getId());
-        this.permissionCheckResult =
-                new DatabasePermissionCheckResult(unauthorizedDatabaseNames);
-    }
-
-    private void doSqlCheck(TaskEntity preCheckTaskEntity, ConnectionConfig connectionConfig,
-            RiskLevelDescriber describer) {
-        if (Objects.isNull(this.databaseChangeRelatedSqls)) {
-            return;
+    private void doSqlCheckAndDatabasePermissionCheck(TaskEntity preCheckTaskEntity, RiskLevelDescriber describer) {
+        List<OffsetString> sqls = new ArrayList<>();
+        this.overLimit = getSqlContentUntilOverLimit(sqls, sqlCheckProperties.getMaxSqlContentBytes());
```

Comment:
sql-check will not get any limit, the caller will control it

Reply:
Yeah, I change the class name to `PreCheckTaskProperties` and move it to package com.oceanbase.odc.service.flow.task.model